tests/integration/test_charm.py

```diff
@@ -23,24 +25,45 @@
 async def test_build_and_deploy(
     ops_test: OpsTest,
     charm_base: str,
```

Comment:
Are we still using `charm_base` here? Don't see any mention of it in this test function. If we don't need it anymore, we should likely remove it as an argument until we need it again.

Reply:
Added the charm base back.